BookingsApi/BookingsApi.IntegrationTests/Api/V1/JudiciaryParticipants/AddJudiciaryParticipantsTests.cs

```diff
@@ -0,0 +1,263 @@
+using BookingsApi.Contract.V1.Requests;
+using BookingsApi.Contract.V1.Requests.Enums;
+using BookingsApi.Contract.V1.Responses;
+using BookingsApi.DAL.Queries;
+using BookingsApi.Validations.V1;
+
+namespace BookingsApi.IntegrationTests.Api.V1.JudiciaryParticipants
+{
+    public class AddJudiciaryParticipantsTests : ApiTest
+    {
+        private readonly string _personalCodeJudge;
+        private readonly string _personalCodePanelMember;
+
+        public AddJudiciaryParticipantsTests()
```

Comment:
Should be treated as a Setup rather than in the ctor?

Reply:
thanks, changed